.github/workflows/helm-check-docs.yaml

```diff
@@ -0,0 +1,13 @@
+name: helm-check-docs
+
+on: push
```

Comment:
Should probably also check on pull requests. Might also be good to only run this workflow when changes happen to files under the `charts` directory.

Reply:
I could be an optim indeed.

Reply:
Done.